src/components/FieldWrapper/FieldWrapper.js

```diff
@@ -49,11 +49,14 @@ FieldWrapper.propTypes = {
     onClear: PropTypes.func,
     /** to style the field as it is focused */
     isFocused: PropTypes.bool,
+    /** defines if the component is disabled */
+    disabled: PropTypes.bool,
 };
 
 FieldWrapper.defaultProps = {
     showClearButton: false,
     clearLabel: '',
     onClear: null,
     isFocused: false,
+    disabled: false,
```

Comment:
Should add this prop to the story as well

Reply:
I didn't add that on purpose since it doesn't change a lot. It just makes FieldWrapper borders disabled and the component it wraps remains active. Maybe it would be better to rework the representative component so we can have disabling option which works for both the wrapper and the child component?

Reply:
What do you mean by "representative component"?

Reply:
I mean FieldWrapper component we have on the storybook page

Reply:
I think storybook should be kept as simple as possible. If we write complex implementations even for basic components, it won't be representative. I think it is ok to just add it as a knob, so one can see what the styling of the disabled component looks like